css/style.css

```diff
@@ -0,0 +1,264 @@
+  background: #0699A6;
+  padding: 22px 64px;
+  font-size: 1.2rem;
+}
+
+
+/* MEDIA QUERIES */
+@media (max-width: 280px){
+  .vector{
+    left: 200px;
+    top: 50px;
+  }
+}
+/* TABLETS */
+@media (min-width: 481px) {
+  header{
+    height: 800px;
+  }
+  .head .head-upper >div .avatar{
+    border-radius: 50%;
+    width: 230px;
+    height: 220px;
+    margin-left: 0px;
+  }.role{
+    margin-top: 48px;
+    margin-left: 10px;
+    font-size: 1.2rem;
+  }
+  .head .head-lower h1{
+    margin-top: 12px!important;
+    font-size: 90px!important;
+    line-height: 80px;
+  }
+  .head .head-lower .contact{
+    padding: 20px 70px;
+    font-size: 1.4rem;
+  }
+  .vector{
+    z-index: 3;
+    left: 405px;
+    top: 45px;
+  }
+  .about-me{
+    margin-top: 0px;
+    padding: 150px;
+    padding-top: 75px;
+    padding-bottom: 40px;
+  }
+}
+
+/* BIGGER */
+@media (min-width: 769px) {
+  header{
+    height: 550px;
+    background: rgba(158, 140, 108, 0.2);
+    color: #fff;
+    position: relative; 
+    padding: 100px 110px;
+    display: block;
+  }
+  .head{
+    display: flex;
+    align-items: center;
+    /* justify-content: space-between; */
+  }
+  .head-upper{
+    margin-right: 90px!important;
+  }  
+  .head .head-upper >div .avatar{
+    border-radius: 50%;
+    width: 295px;
+    height: 295px;
+    margin-left: 0px;
+  }
+  .vector{
+    width: 120px;
+    height: 120px;
+    z-index: 3;
+    left: 315px;
+    top: 285px;
+  }
+  .role{
+    font-size: 14px;
+    margin-top: 0px;
+    margin-left: 0px;
+  }
+  .head .head-lower{
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+   }
+  .head .head-lower h1{
+    margin-top: 0px!important;
+    color: black;
+    font-size: 10rem;
+    margin-bottom: 25px;
+    text-align: start;
+    line-height: 80px;
+  }
+  .head .head-lower .contact{
+    padding: 15px 30px;
+    font-size: 1.0rem;
+  }
+  header::before,
+  header::after {
+    transform: skewY(-3deg);
+    -webkit-transform: skewY(-4.2deg);
+  }
+}
+
+/* 1025px — 1200px: Desktops, large screens */
+@media (min-width: 1025px) {
+  header{
+    height: 580px;
+    background: rgba(158, 140, 108, 0.2);
+    color: #fff;
+    position: relative; 
+    padding: 100px 130px;
+    display: block;
+  }
+  .head{
+    display: flex;
+    align-items: center;
+    /* justify-content: space-between; */
+  }
+  .head-upper{
+    margin-right: 110px!important;
+    margin-left: 70px;
+  }  
+  .head .head-upper >div .avatar{
+    border-radius: 50%;
+    width: 295px;
+    height: 295px;
+    margin-left: 0px;
+  }
+  .vector{
+    width: 120px;
+    height: 120px;
+    z-index: 3;
+    left: 405px;
+    top: 300px;
+  }
+  /* .role{
+    font-size: 20px;
+    margin-top: 0px;
+    margin-left: 0px;
+  } */
+  .head .head-lower{
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    margin-top: 30px;
+   }
+  .head .head-lower h1{
+    margin-top: 10px!important;
```

Comment:
I'm not saying`!important` is bad lol, but if you write your code correctly, you should almost never need it.